pianobar-mediaplayer2

```diff
@@ -15,8 +16,9 @@ class pianoPlayer(mediaplayer.mediaPlayer):
         self.properties["org.mpris.MediaPlayer2.Player"]["CanPlay"] = True
         self.properties["org.mpris.MediaPlayer2.Player"]["CanPause"] = True
         self.properties["org.mpris.MediaPlayer2.Player"]["CanControl"] = True
+        self.properties["org.mpris.MediaPlayer2.Player"]["CanGoNext"] = True
         self.properties["org.mpris.MediaPlayer2"]["CanQuit"] = True
-        self.properties["org.mpris.MediaPlayer2"]["Identity"] = "PianoBar"
+        self.properties["org.mpris.MediaPlayer2"]["Identity"] = "pianobar"
```

Comment:
why was this change necessary?

Reply:
No :upside_down_face: But the official spelling is lowercase, so I figured I'd use that.

Reply:
Sounds good then